docs/snippets/svelte/component-story-static-asset-with-import.js.mdx

```diff
@@ -0,0 +1,24 @@
+```js
+//MyComponent.stories.js
+
+import MyComponent from './MyComponent.svelte';
+
+import imageFile from './static/image.png';
+
+export default {
+  component: MyComponent,
+  title: 'img',
+};
+
+const image = {
+  src: imageFile,
+  alt: "my image",
+};
+
+export const WithAnImage = () => ({
+  Component: MyComponent,
+  props: {
+    image:image
```

Comment:
Lots of spacing issues like this

Reply:
Let me take another pass and I'll commit them. It seems that with each update my VSCode forgets how to format the files. 😦

Reply:
@shilman just pushed some changes. Let me know if you're ok and we'll merge this.